addons/binding/org.openhab.binding.network/ESH-INF/thing/thing-types.xml

```diff
@@ -3,71 +3,118 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
         xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--Network Binding-->
-    <thing-type id="device">
```

Comment:
That's true. Most people use the "ping" device, I propose therefore to add a "device" think mapping to "pingdevice" in the handler factory. It will break for people who are using the "port" parameter, but it was a bad idea in the first place to combine this in one thing. There was an issue I linked to in the PR description: When you use the port parameter, all ping parameters are not used at all. That confused people in the forum and we really should fix this now than later.